### PR TITLE
feat: define source model

### PR DESCRIPTION
**Pull request by NagarajNidagundi**

### CODE REVIEW

File: src/WWTelematics.SourceService/Models/Source.cs [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+        /// <summary>
+        /// Gets or sets a TenantSource of the entity.
+        /// </summary>
+        [JsonProperty(PropertyName = "tenant_source")]
+        [Column(TypeName = "jsonb")]
+        public string TenantSource { get; set; }
+
+        /// <summary>
+        /// Gets or sets a DeviceConfiguration of the entity.
```
Comment (AmitkumarEvry) [inline src/WWTelematics.SourceService/Models/Source.cs]: change DeviceConfiguration  to device_configuration

File: src/WWTelematics.SourceService/Models/Source.cs [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        public Guid PublicId { get; set; }
+
+        /// <summary>
+        /// Gets or sets a label of the entity.
+        /// </summary>
+        [JsonProperty(PropertyName = "label")]
+        public string Label { get; set; }
+
+        /// <summary>
+        /// Gets or sets a TenantSource of the entity.
```
Comment (AmitkumarEvry) [inline src/WWTelematics.SourceService/Models/Source.cs]: change TenantSource  to tenant_source.

File: src/WWTelematics.SourceService/Models/Source.cs (hunk 1 above)
Comment (NagarajNidagundi) [reply src/WWTelematics.SourceService/Models/Source.cs]: done

**Comment by NagarajNidagundi** [comment]
@davidw-foxtrax - Please review this PR

**Pull request merged**